compaction/compaction.cc

```diff
@@ -712,6 +712,7 @@ class compaction {
                 using compact_mutations = compact_for_compaction<compacting_sstable_writer, GCConsumer>;
                 auto cfc = make_stable_flattened_mutations_consumer<compact_mutations>(*schema(), now,
                                          max_purgeable_func(),
+                                         max_repair_purgeable_func(),
```

Comment:
Can we merge this into `max_purgeable_func()`?

Reply:
Maybe not because they'll operate on different clocks.

Reply:
I thought about mergeing them too. But for now, I will keep them seprate: not to mess up the existing logic.